lib/power_action_utils.pm

```diff
@@ -276,14 +276,21 @@ sub power_action {
             reboot_x11;
         }
         elsif ($action eq 'poweroff') {
-            if (is_backend_s390x) {
+            if (is_backend_s390x && check_var("DESKTOP", "gnome")) {
+                send_key "ctrl-alt-delete";
+                assert_screen 'logoutdialog', 15;
+                assert_and_click 'gnome-shell_shutdown_btn';
+                if (get_var("SHUTDOWN_NEEDS_AUTH")) {
+                    assert_screen 'shutdown-auth';
+                    type_password;
+                    send_key "ret" && return 1;
+                }
+            }
+            else {
                 record_info('poo#114439', 'Temporary workaround, because shutdown module is marked as failed on s390x backend when shutting down from GUI.');
                 select_console 'root-console';
                 enter_cmd "$action";
             }
-            else {
-                poweroff_x11;
-            }
```

Comment:
@Zaoliang 
I noticed that you remove 'poweroff_x11' totally. I am wondering if it will impact the power_off action for other platforms [x86_64 aarch64 power s390x_zkvm etc]

Reply:
let me check ;)

Reply:
zkvm is not impacted, this is only related to zVM, gui or console,  see 'else' after 'if (is_backend_s390x && check_var("DESKTOP", "gnome"))' 
I've checked already tha kvm is fine, will provide latest test run anyway, thanks!

Reply:
I implemented actually the replacement of poweroff_x11 because it doesn't work for gui shutdown on zVM.

Reply:
https://openqa.suse.de/tests/8752474 take this job as an example on x86_64 platform.
I think it may change the current logic to shutdown the VM.

Reply:
https://openqa.suse.de/tests/9230121 shows that zkvm test is successful.